chaincode/src/services/PublicKeyService.ts

```diff
@@ -79,7 +79,8 @@ export class PublicKeyService {
 
     // check if we want the profile of the admin
     if (process.env.DEV_ADMIN_PUBLIC_KEY !== undefined && process.env.DEV_ADMIN_USER_ID !== undefined) {
```

Comment:
Could we have to check for falsy values instead of undefined? In our deploy pipeline we have to include the DEV_ADMIN_PUBLIC_KEY and DEV_ADMIN_USER_ID values, but if it is not set then it makes it an empty string. Internally we ran into an issue where because the values were `""` and not undefined it triggered this block and blocked users from being registered with basic-asset.